new_file.txt

```diff
@@ -0,0 +1,3 @@
+this is new file created from feature branch
+
+new code added
```

Comment:
change this to something else but keep commits to one, right now there are two

Reply:
done